src/goto-programs/lazy_goto_model.h

```diff
@@ -46,7 +136,14 @@ class lazy_goto_modelt : public abstract_goto_modelt
   }
 
   /// Create a lazy_goto_modelt from a object that defines function/module pass
-  /// handlers
+  /// handlers.
+  /// Its members functions will be used to construct the lazy model:
+  /// `post_process_function` gets `THandler::process_goto_function`
```

Comment:
Shouldn't it be `handler.process_goto_function`? 
same thing for the other `THandler` below